### PR TITLE
feat: new index

### PR DESCRIPTION
**Pull request by c-ehrlich**
## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-08-15).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

New index page in the style of the new docs. Only made one version for now (all packages, next branch) but should be enough to discuss the draft. 

---

## Screenshots

image

💯

### CODE REVIEW

File: cli/template/addons/tailwind/tailwind.config.cjs [hunk 1]
```diff
@@ -2,7 +2,21 @@
 module.exports = {
   content: ["./src/**/*.{js,ts,jsx,tsx}"],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
```
Comment (juliusmarminge) [inline cli/template/addons/tailwind/tailwind.config.cjs:6]: Not sure if we want to ship this, seems like something everyone would rip out. Are normal TW colors not "close-enligt"?

Comment (c-ehrlich) [reply cli/template/addons/tailwind/tailwind.config.cjs:6]: the light ones are fine, but `purple-900` etc dont go dark enough. could just hardcode some colors inline.

Comment (nexxeln) [reply cli/template/addons/tailwind/tailwind.config.cjs:6]: > the light ones are fine, but `purple-900` etc dont go dark enough. could just do `bg-[#abcdef]` etc where necessary.

I would prefer this yeah. We don't wanna configure anything.

Comment (c-ehrlich) [reply cli/template/addons/tailwind/tailwind.config.cjs]: using all stock colors now except for the background (darker than purple-900)

File: cli/template/page-studs/index/with-auth-trpc-tw.tsx [hunk 2]
```diff
@@ -86,29 +68,162 @@ const AuthShowcase: React.FC = () => {
...
+      <Image
+        src="/nextauth.png"
+        className="h-10 w-8 py-1"
+        alt="NextAuth.js"
+        width="100"
+        height="100"
+      />
+    ),
+  },
+];
```
Comment (juliusmarminge) [inline cli/template/page-studs/index/with-auth-trpc-tw.tsx:229]: Can we try minimizing it a bit? I like having similar theming on the two but I think this is too many lines to scroll when you wanna rip it out ;)

File: cli/template/page-studs/index/with-auth-trpc-tw.tsx [hunk 3]
```diff
@@ -86,29 +68,162 @@ const AuthShowcase: React.FC = () => {
...
+    ),
+  },
+  {
+    name: "NextAuth.js",
+    description:
+      "When you need a flexible, secure, and scalable authentication solution, NextAuth.js is the go-to. It ties into your existing database and provides a simple API to manage users and sessions, and ties into NextJS and tRPC to provide a seamless experience.",
+    documentation: "https://next-auth.js.org/getting-started/introduction",
+    icon: (
+      <Image
+        src="/nextauth.png"
```
Comment (juliusmarminge) [inline cli/template/page-studs/index/with-auth-trpc-tw.tsx:221]: Probably shouldn't ship pngs? Can just remove all icons?

File: cli/template/page-studs/index/with-auth-trpc-tw.tsx [hunk 4]
```diff
@@ -86,29 +68,162 @@ const AuthShowcase: React.FC = () => {
   );
 };
 
-type TechnologyCardProps = {
```
Comment (juliusmarminge) [inline cli/template/page-studs/index/with-auth-trpc-tw.tsx:89]: Why'd you move this down

File: cli/template/page-studs/index/with-auth-trpc-tw.tsx [hunk 5]
```diff
@@ -13,47 +15,27 @@ const Home: NextPage = () => {
...
+      <main className="flex flex-col items-center justify-center min-h-screen bg-[#250657]">
+        <div className="container flex flex-col gap-6 items-center justify-center p-4 mx-auto ">
+          <h1 className="text-5xl md:text-[5rem] font-extrabold text-white tracking-teight">
+            Create <span className="text-purple-300">T3</span> App
+          </h1>
+          <div className="sm:space-y-0 grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-y-4 sm:gap-x-6 sm:gap-y-6 lg:gap-x-8 lg:gap-y-8">
+            {technologyCards.map((technology, idx) => (
+              <TechnologyCard
+                key={`techcard-${idx}`}
+                name={technology.name}
```
Comment (juliusmarminge) [inline cli/template/page-studs/index/with-auth-trpc-tw.tsx:27]: nit: minifiy by spreading props {...technology}

File: cli/template/page-studs/index/with-auth-trpc-tw.tsx (hunk 2 above)
Comment (c-ehrlich) [reply cli/template/page-studs/index/with-auth-trpc-tw.tsx:106]: yeah i think you're right, trimmed it by 80 lines :)

**Comment by c-ehrlich** [comment]
closing this because the branch is super out of data, will open a new version in a minute

**Pull request closed without merging**